test/net_http_test.rb

```diff
@@ -105,6 +105,49 @@ def test_get_is_protected
     end
   end
 
+  def test_get_with_no_bulkhead
```

Comment:
Is there an inspect API in Semian where you can see that no tickets are acquired and no resource was created perhaps?